### ISSUE TITLE
Run the prod publishing-bot as a periodic job

### ISSUE THREAD

**Issue by nikhita** [kubernetes/publishing-bot#353]
https://github.com/kubernetes/test-infra/pull/29118 added a presubmit to run a dry-run of the bot.

We should do something similar to create a periodic job to run the prod bot. That will allow us to have testgrid integrations, easy access to logs and a multitude of other benefits.

**Comment by akhilerm** [comment]
@ameukam Is it possible to get a persistent volume on the build cluster ?

**Comment by ameukam** [comment]
@akhilerm Yes. I'll work on that this week. Any suggestion about size for the PV ? we can start with 50 GB ?

**Comment by akhilerm** [comment]
yep. that would work.  The current production instance uses 100Gi volume as mentioned in https://github.com/kubernetes/publishing-bot/blob/master/artifacts/manifests/pvc.yaml#L4

**Comment by akhilerm** [comment]
@ameukam Gentle reminder to look into this issue for getting PV in test infra

**Comment by sayanchowdhury** [comment]
> we will need at least:
> 
> * a Github token
> * a dedicated volume in the build cluster or a ephemeral one on each run.

@ameukam we would also need new secret with a Github token in the k8s.io, right? Should I file a PR for that in the k8s.io repo?

**Comment by xmudrii** [comment]
@akhilerm What's the status of this issue? Does publishing-bot support running it as a periodic job?
